insights/client/client.py

```diff
@@ -76,7 +76,9 @@ def set_up_logging():
         config['to_stdout'] = (config['to_stdout'] or config['from_stdin'] or config['from_file'])
 
         verbose = config['verbose'] or config['verbose']
-        logging.root.addHandler(get_console_handler(config['silent'], verbose, config['to_stdout']))
+        logging.root.addHandler(get_console_handler(config['silent'],
+                                verbose,
```

Comment:
Please indent to the inner parens.  It looks like these last two arguments are passed to `addHandler`.